Wbooks/src/redux/store.js

```diff
@@ -8,9 +9,11 @@ const reducers = {
   bookReducer
 };
 
+// Reactotron.createEnhancer(true)
+
```

Comment:
how can I add thunk and this at the same time?

Reply:
Oh, ok. import `compose`from redux and try this
```
const enhancers = [];
enhancers.push(applyMiddleware(thunk));
if (__DEV__) enhancers.push(Reactotron.createEnhancer(true));
const store = createStore(appReducer, compose(...enhancers));
```